swarm/swarm.go

```diff
@@ -214,7 +214,7 @@ func NewSwarm(config *api.Config, mockStore *mock.NodeStore) (self *Swarm, err e
 	tags := chunk.NewTags() //todo load from state store
 
 	// Swarm Hash Merklised Chunking for Arbitrary-length Document/File storage
-	self.fileStore = storage.NewFileStore(self.netStore, self.config.FileStoreParams, tags)
+	self.fileStore = storage.NewFileStore(localStore, self.config.FileStoreParams, tags)
```

Comment:
can you elaborate about the reason for this change?

Reply:
Local insertion of chunks goes through the network layer checks (i.e. chunk validation, network lock)  today.  These checks make are redundant for local chunks and make the upload slow. So i have short circuited the network layer here to directly push the chunks to localstore. This improves the upload speed by 50%.

Reply:
May be we should take this as part of another effort to reduce upload speed. what do you think?

Reply:
I agree. I think it’s better we address this separately. I’d also want to do some benchmarking in this regard. Nice catch!

Reply:
yeah.. I agree. Removing this from this commit

Reply:
@jmozah @justelad is right we should keep this as `self.netStore` - the reason for this is that `self.fileStore` is used by the LazyChunkReader, so if a chunk is not there, you need the network layer.